src/adapter/src/explain_new/mir/text.rs

```diff
@@ -125,6 +125,9 @@ impl<'a> Displayable<'a, MirRelationExpr> {
                     })?;
                 }
             }
+            MirRelationExpr::LetRec { .. } => {
```

Comment:
Ah, because I don't understand the `Let` case above. Perhaps it is just that easy though!

Reply:
(( the HIR case was just a lot simpler to grok, and I didn't need to understand if we needed prep work for the linear chains, what that `while` loop is doing, etc. ))

Reply:
I'll add that case once we merge this.

Reply:
I have this locally fwiw. It was pretty easy, but needs something like `NormalizeLets` to avoid panicking in the normalization part of `EXPLAIN`. It all seems to work, but wants the new `NormalizeLets` that I am holding back for fear of complicating the PR. :D